ecl/compute/v2/_proxy.py

```diff
@@ -816,3 +817,15 @@ def delete_volume(self, volume, ignore_missing=False):
         :returns: ``None``
         """
         self._delete(_volume.Volume, volume, ignore_missing=ignore_missing)
+
+    def retype_volume(self, volume, volume_type, is_dry_run=False):
+        """Retype an volume
+
+        :param volume:The value can be either the ID of an volume or a
+                      :class:`~ecl.compute.v2.volume.Volume` instance.
+        :param volume_type: volume type of volume to create.
+        :param bool is_dry_run:
+        :returns: :class:`~ecl.compute.v2.server_interface.ServerInterface`
```

Comment:
```suggestion
        :returns: <Response 202>
```
同じ _proxy.pyの resize_server関数などでは戻り値がこうなっています。

Reply:
おっしゃる通りに修正いたしました。